CHANGELOG.asciidoc

```diff
@@ -44,6 +44,7 @@ Use subheadings with the "=====" level for adding notes for unreleased changes:
 * Capture bucket and object key to Lambda transaction as OTel attributes - `aws.s3.bueckt`, `aws.s3.key` - {pull}3364[#3364]
 * Added `context_propagation_only` configuration option - {pull}3358[#3358]
 * Added attribute[*] for JMX pattern metrics (all metrics can now be generated with `object_name[*:type=*,name=*] attribute[*]`) - {pull}3376[#3376]
+* Only automatically apply redacted exceptions for Corretto JVM 17-20. Outside that, user should use capture_exception_details=false to workaround the JVM race-condition bug if it gets triggered: {pull}3438[#3438]
```

Comment:
That is also the wrong place, this is for the already released 1.44.0 version. After a merge/rebase from main you'll have the new `Features` section iund `=== Unreleased` from #3406 .

Reply:
LOL I'm in such a hurry that I am making it slower for me!

Reply:
okay, did it right this time, sorry for the dumbness